<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1101</width>
    <height>630</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1101</width>
    <height>630</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1101</width>
    <height>630</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>763</width>
      <height>571</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>763</width>
      <height>571</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>763</width>
      <height>571</height>
     </size>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="6">
      <widget class="QLabel" name="label_other_2">
       <property name="minimumSize">
        <size>
         <width>177</width>
         <height>91</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>177</width>
         <height>91</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="8">
      <widget class="QPushButton" name="pushButton_next">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="text">
        <string>后一张</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2" colspan="5">
      <widget class="QLabel" name="label_show">
       <property name="minimumSize">
        <size>
         <width>585</width>
         <height>401</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>585</width>
         <height>401</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="lineWidth">
        <number>1</number>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="0" column="7">
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="4">
      <widget class="QLabel" name="label_other_1">
       <property name="minimumSize">
        <size>
         <width>177</width>
         <height>91</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>177</width>
         <height>91</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::WinPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="6">
      <widget class="QCheckBox" name="checkBox">
       <property name="minimumSize">
        <size>
         <width>61</width>
         <height>31</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>61</width>
         <height>31</height>
        </size>
       </property>
       <property name="text">
        <string>水印</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="pushButton_last">
       <property name="minimumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>64</width>
         <height>64</height>
        </size>
       </property>
       <property name="text">
        <string>前一张</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QLabel" name="label_other_0">
       <property name="minimumSize">
        <size>
         <width>177</width>
         <height>91</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>177</width>
         <height>91</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">border:2px solid red;</string>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item row="3" column="4">
      <widget class="QPushButton" name="pushButton_save">
       <property name="minimumSize">
        <size>
         <width>151</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>151</width>
         <height>28</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>保存图片</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="1" column="5">
      <spacer name="horizontalSpacer_4">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="3" column="2">
      <widget class="QPushButton" name="pushButton_change">
       <property name="minimumSize">
        <size>
         <width>151</width>
         <height>28</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>151</width>
         <height>28</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>处理同步</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QDockWidget" name="dockWidget_2">
   <property name="minimumSize">
    <size>
     <width>163</width>
     <height>314</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>163</width>
     <height>314</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>基础功能</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_2">
    <widget class="QLabel" name="label_binary">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>80</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>128</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>饱和度：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>对比度：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>二值化：</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_light">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::add-page:Horizontal
     {     
        background-color: rgb(223, 223, 223);
        height:4px;
     }
     QSlider::sub-page:Horizontal 
    {
        background-color:black;
        height:4px;
     }
    QSlider::groove:Horizontal 
    {
        background:transparent;
        height:10px;
		border-radius: 5px;
    }
    QSlider::handle:Horizontal 
    {
        height: 20px;
        width:20px;
        border-image: url();
        margin: -8 0px;
    }
</string>
     </property>
     <property name="minimum">
      <number>-255</number>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_light">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>调节亮度值：</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_contrast">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>180</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::add-page:Horizontal
     {     
        background-color: rgb(223, 223, 223);
        height:4px;
     }
     QSlider::sub-page:Horizontal 
    {
        background-color:black;
        height:4px;
     }
    QSlider::groove:Horizontal 
    {
        background:transparent;
        height:10px;
		border-radius: 5px;
    }
    QSlider::handle:Horizontal 
    {
        height: 20px;
        width:20px;
        border-image: url();
        margin: -8 0px;
    }
</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_saturability">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::add-page:Horizontal
     {     
        background-color: rgb(223, 223, 223);
        height:4px;
     }
     QSlider::sub-page:Horizontal 
    {
        background-color:black;
        height:4px;
     }
    QSlider::groove:Horizontal 
    {
        background:transparent;
        height:10px;
		border-radius: 5px;
    }
    QSlider::handle:Horizontal 
    {
        height: 20px;
        width:20px;
        border-image: url();
        margin: -8 0px;
    }
</string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_binary">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::add-page:Horizontal
     {     
        background-color: rgb(223, 223, 223);
        height:4px;
     }
     QSlider::sub-page:Horizontal 
    {
        background-color:black;
        height:4px;
     }
    QSlider::groove:Horizontal 
    {
        background:transparent;
        height:10px;
		border-radius: 5px;
    }
    QSlider::handle:Horizontal 
    {
        height: 20px;
        width:20px;
        border-image: url();
        margin: -8 0px;
    }
</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>128</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget_3">
   <property name="minimumSize">
    <size>
     <width>163</width>
     <height>255</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>163</width>
     <height>255</height>
    </size>
   </property>
   <property name="floating">
    <bool>false</bool>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="allowedAreas">
    <set>Qt::AllDockWidgetAreas</set>
   </property>
   <property name="windowTitle">
    <string>色彩调整</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents_3">
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>12</x>
       <y>90</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>G值：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>160</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>B值：</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_G">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>120</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::add-page:Horizontal
     {     
        background-color: white;
        height:4px;
     }
     QSlider::sub-page:Horizontal 
    {
        background-color:green;
        height:4px;
     }
    QSlider::groove:Horizontal 
    {
        background:transparent;
        height:10px;
		border-radius: 5px;
    }
    QSlider::handle:Horizontal 
    {
        height: 20px;
        width:20px;
        border-image: url();
        margin: -8 0px;
    }

</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_R">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="contextMenuPolicy">
      <enum>Qt::DefaultContextMenu</enum>
     </property>
     <property name="toolTipDuration">
      <number>1</number>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::add-page:Horizontal
     {     
        background-color: white;
        height:4px;
     }
     QSlider::sub-page:Horizontal 
    {
        background-color:red;
        height:4px;
     }
    QSlider::groove:Horizontal 
    {
        background:transparent;
        height:10px;
		border-radius: 5px;
    }
    QSlider::handle:Horizontal 
    {
        height: 20px;
        width:20px;
        border-image: url();
        margin: -8 0px;
    }

</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="tickPosition">
      <enum>QSlider::NoTicks</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>11</x>
       <y>20</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>R值：</string>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_B">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>190</y>
       <width>141</width>
       <height>22</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::add-page:Horizontal
     {     
        background-color: white;
        height:4px;
     }
     QSlider::sub-page:Horizontal 
    {
        background-color:blue;
        height:4px;
     }
    QSlider::groove:Horizontal 
    {
        background:transparent;
        height:10px;
		border-radius: 5px;
    }
    QSlider::handle:Horizontal 
    {
        height: 20px;
        width:20px;
        border-image: url();
        margin: -8 0px;
    }

</string>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockWidget">
   <property name="minimumSize">
    <size>
     <width>172</width>
     <height>569</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>172</width>
     <height>569</height>
    </size>
   </property>
   <property name="features">
    <set>QDockWidget::DockWidgetMovable</set>
   </property>
   <property name="windowTitle">
    <string>工具箱</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <widget class="QPushButton" name="pushButton_file">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>选择图片</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_ratio">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>290</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>0:1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>边缘比例：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_edge_org">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>原图边缘检测</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_gamma">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>320</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>伽马变换</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_t_left">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>左旋</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSlider" name="horizontalSlider_ratio">
     <property name="geometry">
      <rect>
       <x>9</x>
       <y>260</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QSlider::add-page:Horizontal
     {     
        background-color: rgb(223, 223, 223);
        height:4px;
     }
     QSlider::sub-page:Horizontal 
    {
        background-color:black;
        height:4px;
     }
    QSlider::groove:Horizontal 
    {
        background:transparent;
        height:10px;
		border-radius: 5px;
    }
    QSlider::handle:Horizontal 
    {
        height: 20px;
        width:20px;
        border-image: url();
        margin: -8 0px;
    }
</string>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="singleStep">
      <number>1</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_t_level">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="text">
      <string>水平
变换</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_t_right">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>370</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="text">
      <string>右旋</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_edge">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>边缘检测</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_t_vertical">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>450</y>
       <width>64</width>
       <height>64</height>
      </rect>
     </property>
     <property name="text">
      <string>垂直
变化</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_average">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>140</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>均值滤波</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_origin">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>显示原图</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_gray">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>151</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>灰度化</string>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <zorder>dockWidget_2</zorder>
  <zorder>dockWidget_3</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
